code/modules/chemistry/Reagents-Misc.dm

```diff
@@ -521,17 +521,24 @@ datum
 			depletion_rate = 0.2
 			value = 28 // 3 3 22
 			viscosity = 0.5
+			on_add()
+				..()
+				if(ismob(src.holder?.my_atom))
+					RegisterSignal(holder.my_atom, COMSIG_MOB_SHOCKED_DEFIB, .proc/revive)
```

Comment:
feel like tasers should work

Reply:
or any electric shock

Reply:
working on it, almost got defibrillators and tasers to work

Reply:
yass says they dont want tasers to work on it so